build-utils/language-go/src/test/resources/plc4go/pom.xml

```diff
@@ -244,75 +225,61 @@
         <!--argLine>@{failsafeArgLine}</argLine-->
       </configuration>
     </plugin>
-
-
   </plugins>
 </build>
 
 <dependencies>
   <dependency>
     <groupId>org.apache.plc4x</groupId>
     <artifactId>plc4j-api</artifactId>
-    <version>0.9.0-SNAPSHOT</version>
+    <version>${project.version}</version>
```

Comment:
Please leave this at the normal version not a variable.

Reply:
Changed back.